upgrade/1.2/scripts/rebuild/prerequisites.sh

```diff
@@ -57,4 +57,36 @@ else
     echo "====> ${state_name} has been completed"
 fi
 
+state_name="ELIMINATE_NTP_CLOCK_SKEW_M001"
+state_recorded=$(is_state_recorded "${state_name}" "${target_ncn}")
+if [[ $state_recorded == "0" && $2 != "--rebuild" ]]; then
+    echo "====> ${state_name} ..."
+    {
+    if [[ "${target_ncn}" == "ncn-m001" ]]; then
+      # ensure the directory exists
+      mkdir -p /srv/cray/scripts/common/
+      # copy the ntp script and template to ncn-m001
+      cp -r "${CSM_ARTI_DIR}"/chrony/ /srv/cray/scripts/common/
```

Comment:
Should `-p` or `--preserve=mode` be used to preserve execute permission?

```suggestion
      cp -r --preserve=mode "${CSM_ARTI_DIR}"/chrony/ /srv/cray/scripts/common/
```